snapfaas/src/sched/messages.proto

```diff
@@ -0,0 +1,61 @@
+syntax = "proto3";
+
+package snapfaas.sched.messages;
+
+// Begin of request messages
+message GetJob {
+    uint64 id = 1;
+}
+
+message FinishJob {
+    string id     = 1;
+    bytes  result = 2;
+}
+
+message Invoke {
+    bytes invoke = 1;
```

Comment:
Is there not defined shape to this request? What is contained in these bytes? (presumably a function name and arguments) Can it be borrowed/shared with the system call protocol buffer and hopefully avoid unnecessary copying _or_ unnecessary reliance on hidden invariants?

Reply:
It's the serialized `LabeledInvoke`. The syscall proto `invoke` doesn't have the label field, and I found it convenient to just serialize `LabeledInvoke` and pass it around. Should I break it down to gate, payload, and label as in `LabeledInvoke`?